### PR TITLE
Update Homebrew script

### PR DESCRIPTION
**Pull request by andreiborisov**
Update link to the core Homebrew repo used during installation. Add an option to install using shallow clone as a feature.

### CODE REVIEW

**Comment by andreiborisov** [comment]
@Chuxel not sure, if the tests are failing due to the update

File: script-library/homebrew-debian.sh [hunk 1]
```diff
@@ -117,12 +117,12 @@ if [ "${USE_SHALLOW_CLONE}" = "false" ]; then
     echo "Installing Homebrew..."
     git clone https://github.com/Homebrew/brew "${BREW_PREFIX}/Homebrew"
     mkdir -p "${BREW_PREFIX}/Homebrew/Library/Taps/homebrew"
-    git clone https://github.com/Homebrew/linuxbrew-core "${BREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
+    git clone https://github.com/Homebrew/homebrew-core "${BREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
```
Comment (joshspicer) [inline script-library/homebrew-debian.sh:120]: Thanks for contributing! Why did you make this change?

Comment (andreiborisov) [reply script-library/homebrew-debian.sh:120]: > Thanks for contributing! Why did you make this change?

Previously, Homebrew used a separate repo for Linux, but now they merged Mac and Linux repos into a single one: https://github.com/Homebrew/linuxbrew-core/blob/master/README.md

Comment (joshspicer) [reply script-library/homebrew-debian.sh:120]: gotcha, thanks.

**Pull request merged**